tderenko/ithillel_mvc
# [homework-24] MVC

# issue
Зробити наступний функціонал:

- Створення категорій
- Створення постів
- Вивід постів і категорій для юзерів

Не забуваємо про меделі та міграції

# Test Plan
1. create migration by command `docker exec ithillel_mvc-php-1 php run.php migrate:create users`
![image](https://user-images.githubusercontent.com/18249681/190870194-8bf6399e-a722-48a3-b22d-105752086968.png)
2. run migration `docker exec ithillel_mvc-php-1 php run.php migrate`
3. go to registration page and fill required fields and submit
4. you should be login
5. go to admin dashboard
![image](https://user-images.githubusercontent.com/18249681/190870309-16358f0d-3319-4445-9947-8525d04bea11.png)
6. go to category and create some categories
![image](https://user-images.githubusercontent.com/18249681/190870340-8eae1239-5718-4678-bb26-b1e2db855a99.png)
7. go to posts and create
8. go to home page then blog and choice some category
![image](https://user-images.githubusercontent.com/18249681/190870395-4d757815-dee8-4e53-9b35-054cd03b5416.png)
9. open some post
![image](https://user-images.githubusercontent.com/18249681/190870412-de79b10a-1bc7-42a7-98cf-eba4c39f2621.png)
10. you should see the result
![image](https://user-images.githubusercontent.com/18249681/190870920-ef9db9c5-d0e1-4705-aae8-3fb9bb970033.png)
